apps/test/unit/storage/dataBrowser/DataVisualizerTest.js

```diff
@@ -12,21 +13,81 @@ const DEFAULT_PROPS = {
 };
 
 describe('DataVisualizer', () => {
-  var wrapper;
-
   it('The modal starts closed', () => {
-    wrapper = shallow(<DataVisualizer {...DEFAULT_PROPS} />);
+    let wrapper = shallow(<DataVisualizer {...DEFAULT_PROPS} />);
     expect(wrapper.find(BaseDialog).prop('isOpen')).to.be.false;
   });
 
   it('The modal opens when the button is clicked', () => {
-    wrapper = shallow(<DataVisualizer {...DEFAULT_PROPS} />);
+    let wrapper = shallow(<DataVisualizer {...DEFAULT_PROPS} />);
     expect(wrapper.find(BaseDialog).prop('isOpen')).to.be.false;
 
     wrapper.instance().handleOpen();
     expect(wrapper.find(BaseDialog).prop('isOpen')).to.be.true;
   });
 
+  describe('updateChart', () => {
+    let wrapper;
+    let spy;
+    beforeEach(() => {
+      GoogleChart.lib = {};
+      spy = sinon.stub(GoogleChart.prototype, 'drawChart');
+      const STARTING_PROPS = {
+        tableColumns: ['category1', 'category2'],
+        tableName: 'testTable',
+        tableRecords: [
+          '{"category1" : "red", "category2": 1}',
+          '{"category1" : "blue", "category2": 1}',
+          '{"category1" : "red", "category2": 3}',
+          '{"category1" : "green", "category2": 4}'
+        ]
+      };
+      wrapper = shallow(<DataVisualizer {...STARTING_PROPS} />);
+      wrapper.instance().handleOpen();
+      var chartArea = document.createElement('div');
+      chartArea.setAttribute('id', 'chart-area');
+      document.body.appendChild(chartArea);
```

Comment:
Note: `afterEach` should probably remove this DOM element too.  The [`enforceDocumentBodyCleanup`](https://github.com/code-dot-org/code-dot-org/blob/e195c719b94c562960fcd6659421d2a97877ddcb/apps/test/util/testUtils.js#L338) helper might be useful here.